app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
- [ ] You have two actions in the UsersController :blush: That's great but you should have two views for these two actions :thinking: As you can see in your `app/views/users` folder that there is only one view which is **index.html.erb** you should have another view by the name of **show.html.erb** :blush: for your show action :+1: